### PR TITLE
net-im/skypeforlinux: Bump version to 8.56.0.103

### PR DESCRIPTION
**Pull request by jvesely**
Cleanup old

### CODE REVIEW

File: net-im/skypeforlinux/skypeforlinux-8.56.0.103.ebuild [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+	domenu usr/share/applications/skypeforlinux.desktop
+
+	if use pax_kernel; then
+		pax-mark -m "${ED}"/opt/skypeforlinux/skypeforlinux
+		pax-mark -m "${ED}"/opt/skypeforlinux/resources/app.asar.unpacked/node_modules/slimcore/bin/slimcore.node
+		ewarn "You have set USE=pax_kernel meaning that you intend to run"
+		ewarn "${PN} under a PaX enabled kernel. To do so, we must modify"
+		ewarn "the ${PN} binary itself and this *may* lead to breakage! If"
+		ewarn "you suspect that ${PN} is being broken by this modification,"
+		ewarn "please open a bug."
```
Comment (juippis) [inline net-im/skypeforlinux/skypeforlinux-8.56.0.103.ebuild]: We usually pax-mark unconditionally these days. Will it break the program for those not running hardened kernel?

Comment (juippis) [reply net-im/skypeforlinux/skypeforlinux-8.56.0.103.ebuild]: Also, 
```
UnderscoreInUseFlag: USE flag 'pax_kernel' uses reserved underscore character
```

Comment (jvesely) [reply net-im/skypeforlinux/skypeforlinux-8.56.0.103.ebuild]: I can test whether unconditional pax-mark works on my machine.
The same use flag name is used in several packages so I'd expect someone on the security side is handling those. if we decide that keeping the useflag is necessary

Comment (juippis) [reply net-im/skypeforlinux/skypeforlinux-8.56.0.103.ebuild]: The "no underscores in USE flags" is relatively new policy, and should be fixed as packages are bumped.
https://projects.gentoo.org/qa/policy-guide/use-flags.html#underscores-in-use-flag-names

Let me know whether it works or not.

Comment (jvesely) [reply net-im/skypeforlinux/skypeforlinux-8.56.0.103.ebuild]: I've installed -r1 with unconditional pax marking. I joined one video call with several participants, camera, video, and audio worked ok. my mic stopped working after a while, but I'm not sure if that was caused by pax markings, or it was some other issue.
I also noticed that skype no longer blocked keyboard input on my machine, which is a plus.

Comment (jvesely) [reply net-im/skypeforlinux/skypeforlinux-8.56.0.103.ebuild]: I've had a couple more online meetings with unconditional pax markings and it looks ok. There are occasional hangs and audio glitches, but I'd attribute them to network.

**Pull request closed without merging**